src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
     public String getRandomColor() {
-        return null;
+        String color;
+        Random random = new Random();
+        int ranValue = random.nextInt(4);
+        Colors[] allColors = Colors.values();
+        color = allColors[ranValue].toString();
+        return color;
```

Comment:
ranValue -> index
allColors -> colors
4 -> colors.length
```suggestion
        Random random = new Random();
        Colors[] colors = Colors.values();
        int index = random.nextInt(colors.length);
        return colors[index].toString();
```